test/object-store/sync/client_reset.cpp

```diff
@@ -1048,6 +1048,13 @@ TEST_CASE("sync: client reset", "[sync][pbs][client reset][baas]") {
                     },
                     std::chrono::seconds(20), std::chrono::milliseconds(500));
             }
+            // We can't be sure that the 'after' callback has been called yet
+            timed_sleeping_wait_for(
```

Comment:
Why isn't the wait at line 1049 enough? Maybe we should extend `std::chrono::milliseconds(500)` to whatever it is needed.

Reply:
Ok - I thought I removed the 500 ms addition in line 1049. The 500 ms is just the time we sleep between each time we check the condition. If the condition is true, we will return immediately.